CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [CHANGE] Histogram `cortex_memcache_request_duration_seconds` `method` label value changes from `Memcached.Get` to `Memcached.GetBatched` for batched lookups, and is not reported for non-batched lookups (label value `Memcached.GetMulti` remains, and had exactly the same value as `Get` in nonbatched lookups).  The same change applies to tracing spans. #3046
 * [CHANGE] TLS server validation is now enabled by default, a new parameter `tls_insecure_skip_verify` can be set to true to skip validation optionally. #3030
 * [CHANGE] `cortex_ruler_config_update_failures_total` has been removed in favor of `cortex_ruler_config_last_reload_successful`. #3056
+* [FEATURE] Update in dependency `weaveworks/common`. Logging of IP of reverse proxy is now supported. #2985
```

Comment:
My suggestion when writing the CHANGELOG is to put yourself in the final user's shoes. The final user has no idea what `weaveworks/common` is but would love to know the new config options to control the logging of IP. So my suggestion is:
- Remove mentioning to the updated dependency
- Mention the new CLI flags introduced and what they do

Reply:
Good idea. Updated.